nRF Toolbox/Profiles/BloodPressure/BloodPressureTableViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BloodPressureTableViewController.swift
+//  nRF Toolbox
+//
+//  Created by Nick Kibysh on 01/10/2019.
+//  Copyright © 2019 Nordic Semiconductor. All rights reserved.
+//
+
+import UIKit
+import CoreBluetooth
+
+extension Identifier where Value == Section {
+    static let bloodPressure: Identifier<Section> = "bloodPressure"
+    static let cuffPressure: Identifier<Section> = "cuffPressure"
+    static let dateTime: Identifier<Section> = "TimeBloodPressureSection"
+    static let heartRate: Identifier<Section> = "HeartRate"
+}
+
+class BloodPressureTableViewController: PeripheralTableViewController {
+    private var bloodPressureSection = BloodPressureSection(id: .bloodPressure)
+    private var cuffPressureSection = CuffPressureSection(id: .cuffPressure)
+    private var dateTimeSection = TimeBloodPressureSection(id: .dateTime)
+    private var heartRateSection = PulseBloodPressureSection(id: .heartRate)
+    
+    override var internalSections: [Section] { [bloodPressureSection, heartRateSection, dateTimeSection, cuffPressureSection] }
+    override var peripheralDescription: Peripheral { .bloodPressure }
+    
+    private var dataSectionIds: [Identifier<Section>] = [.bloodPressure, .cuffPressure]
+    override var navigationTitle: String { "Blood Pressure Monitor" }
+    
+    override func didUpdateValue(for characteristic: CBCharacteristic) {
+        switch characteristic.uuid {
+        case CBUUID.Characteristics.BloodPressure.measurement:
+            let bloodPressureCharacteristic = BloodPreasureCharacteristic(data: characteristic.value!)
+
+            bloodPressureSection.update(with: bloodPressureCharacteristic)
+            heartRateSection.update(with: bloodPressureCharacteristic)
+            dateTimeSection.update(with: bloodPressureCharacteristic)
+            
+            setBloodPressureVisibility(true)
+            cuffPressureSection.isHidden = true
+            tableView.reloadData()
+        case CBUUID.Characteristics.BloodPressure.intermediateCuff:
+            cuffPressureSection.update(with: CuffPreasureCharacteristic(data: characteristic.value!))
+            setBloodPressureVisibility(false)
+            cuffPressureSection.isHidden = false
+            tableView.reloadData()
+        default:
+            super.didUpdateValue(for: characteristic)
+        }
+    }
+    
+    private func setBloodPressureVisibility(_ isVisible: Bool) {
+        bloodPressureSection.isHidden = !isVisible
+        heartRateSection.isHidden = !isVisible
+        dateTimeSection.isHidden = !isVisible
```

Comment:
Trick: You can write an extension to UIView and define an @isVisible property that does just this, so we don't have to think about so much ! logic.

Also, you can do all of this in one line:
`[bloodPressureSection, heartRateSection, dateTimeSection].forEach { $0.isVisible = isVisible }`

More Swifty ;)

Reply:
Sections are not views so I cannot add extensions. 
That are structs / classes which represent table section data. 
Is hidden is a property from `Section` protocol. So I cannot add simple extension.

But one line statement looks great, thanks.